src/Nav.js

```diff
@@ -81,9 +82,16 @@ function Nav({
           placeholder="Search sound..."
           // type="search" // This seems appropriate, but adds an X that interferes with the search icon.
         />
-        {!isLoggedIn && (
+        {isLoggedIn ? (
+          <LoginDropdown />
+        ) : (
           <Button onClick={openRedirectDialogModalWindow}>Login</Button>
```

Comment:
Can you rename `openRedirectDialogModalWindow` to `openModal`

Reply:
done